cms/templates/home_page.html

```diff
@@ -130,13 +130,13 @@ <h2 class="course-card-title">
   </div>
 {% endif %}
 {% if show_home_page_video_component and page.video_component_title %}
-<div class="home-page-video-row d-flex justify-content-center row">
-  <div class="home-page-video-container">
-    <div class="home-page-video-description">
+<div class="home-page-video-row d-flex justify-content-center container">
```

Comment:
The `.home-page-video-row` class defines "display: flex" which I think can be removed if d-flex is defined here.

Reply:
correct! thank you!